tensorflow/compiler/xla/service/gpu/gpu_conv_algorithm_picker.cc

```diff
@@ -175,12 +590,11 @@ StatusOr<bool> GpuConvAlgorithmPicker::RunOnInstruction(HloInstruction* instr) {
                       instr->backend_config<CudnnConvBackendConfig>());
   backend_config.set_algorithm(best_algo.conv().algorithm());
   backend_config.set_tensor_ops_enabled(best_algo.conv().tensor_ops_enabled());
-  backend_config.set_scratch_size(best_algo.scratch_bytes());
```

Comment:
Line 172 make it possible to remove scratch_size field, because the information is already recorded in the HLO instruction. All we need in `cudnn_conv_algorithm_runner` is to obtain the information from the instruction again.